src/PKSim.Core/Services/TransportContainerUpdater.cs

```diff
@@ -73,6 +73,10 @@ private void updateFractionExpressedEpithelial(TransporterExpressionContainer tr
          if (fractionExpressedApical == null)
             return;
 
+         //Parameter is a hidden parameter. This is used for consistency purpose only and should not be updated
+         if (!fractionExpressedApical.Visible)
```

Comment:
A bit to eager here. The null above was covering exactly this. But now that we have this parameter everywhere, we should only update if we can actually set it

Reply:
lol
always those nice side effects 😄